wiki/en/Running-a-Server.md

```diff
@@ -335,6 +304,9 @@ For macOS, start a Terminal window and run Jamulus with the desired options like
 
 # Troubleshooting
 
-Various problems can arise when setting up Servers, and we generally encourage people to use other people's (public) Servers at first to isolate any issues that may in fact be due to Client configuration.
+If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
```

Comment:
This was mentioned earlier and isn't about running a server.

Reply:
I do feel bad about this... !

Reply:
Yeah. It's just a copy from above. Maybe we should move this to the Server troubleshooting page?

@gilgongo ?

Reply:
The advice to use other people's Servers is to head off any issues later, so it's not really troubleshooting. If we want to say it at all, should be part of the setup really.

Reply:
I've removed it. I think there are valuable practical tips on that page, but probably too many...